client/src/components/Map/Map.jsx

```diff
@@ -94,7 +76,7 @@ class Map extends Component {
       location: place.geometry.location,
       placeId: place.place_id,
       radius: '500',
-      query: "cafe"
+      query: "Cafe"
```

Comment:
What other categories are there? is there a way to also query for specific names and add that to the list that is returned? like Starbucks / McDonalds / Dunkin Donuts etc etc?

Reply:
There is a list of locations here: https://tinyurl.com/yxtm5qns. It is also possible to make the query dynamic based on whatever the user is looking for, you would just need to connect it to the input box.

Reply:
Can the base query do multiple categories? I feel like libraries would be good to include.